### DISCUSSION TITLE
Works with Brother HL-L2445DW

### DISCUSSION THREAD

**Discussion by datacyclist** [Owl-Maintain/brlaser#15] [General]
Just a note on this: I installed the package in gentoo (net-print/brlaser-6-1), restarted cups and chose the HL-2270DW printer in the cups dialogue.

It works nicely with my Brother HL-L2445DW printer.

**Comment by QORTEC** [comment]
@datacyclist Thanks for letting us know that your printer works.

If you are able to; please connect your printer to your computer via USB, and then run the following command: `sudo lpinfo --include-schemes usb -l -v`.
By providing the commands output, I will be able to add a entry for your printer to `brlaser`.
